tests/app/conftest.py

```diff
@@ -241,6 +335,7 @@ def create_sample_template(
     subject_line="Subject",
     user=None,
     service=None,
+    category=None,
```

Comment:
Do we need template_category_id here instead of category?

Reply:
Just setting the `category_id` should work fine. What we could also do is if the category is `None` we can set the `category_id` to one of the default categories.